src/main/java/org/broadinstitute/consent/http/db/mapper/DacWithDaasReducer.java

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.consent.http.db.mapper;
+
+import java.util.Map;
+import java.util.Objects;
+import org.broadinstitute.consent.http.models.Dac;
+import org.broadinstitute.consent.http.models.DataAccessAgreement;
+import org.broadinstitute.consent.http.models.FileStorageObject;
+import org.broadinstitute.consent.http.util.ConsentLogger;
+import org.jdbi.v3.core.mapper.MappingException;
+import org.jdbi.v3.core.result.LinkedHashMapRowReducer;
+import org.jdbi.v3.core.result.RowView;
+
+public class DacWithDaasReducer implements LinkedHashMapRowReducer<Integer, Dac>,
```

Comment:
I can see this class merged with `DacWithDatasetsReducer` and then perhaps renamed to `DacReducer`.